public/components/content-list-drawer/content-list-drawer.html

```diff
@@ -160,31 +160,113 @@
+                                    <a href="{{ usage.webUrl }}" target="_blank"><i class="drawer__icon" wf-icon="live-site"></i></a>
                                 </div>
                             </div>
                         </li>
                     </ul>
+
+                    <ul class="drawer__column">
+                        <li class="drawer__item">
+                            <p class="drawer__item-title">Video source</p>
+                            <span class="drawer__item-content">
+                                {{ capiData.source || '-' }}
+                            </span>
+                        </li>
+
+                        <li class="drawer__item">
+                            <p class="drawer__item-title">Video category</p>
+                            <span class="drawer__item-content">
+                                {{ capiData.category || '-' }}
+                            </span>
+                        </li>
+
+                        <li class="drawer__item">
+                            <p class="drawer__item-title">Expiry date</p>
+                            <span class="drawer__item-content">
+                                {{ capiData.friendlyExpiryDate || '-' }}
+                            </span>
+                        </li>
+
+                    </ul>
+
+                    <ul class="drawer__column">
+                        <li class="drawer__item">
+                            <p class="drawer__item-title">Youtube category</p>
+                            <span class="drawer__item-content">
+                                {{ capiData.metadata.categoryId || '-' }}
+                            </span>
+                        </li>
+
+                        <li class="drawer__item">
+                            <p class="drawer__item-title">Youtube channel</p>
+                            <span class="drawer__item-content">
+                                {{ capiData.metadata.channelId || '-' }}
+                            </span>
+                        </li>
+
+                        <li class="drawer__item">
+                            <p class="drawer__item-title">Keywords</p>
+                            <span class="drawer__item-content">
+                                {{ capiData.metadata.tags || '-' }}
+                            </span>
+                        </li>
+
+                    </ul>
+
+                    <ul class="drawer__column drawer__column--wide" ng-if="capiData.youtubeUrl">
+                        <li class="drawer__item">
+                            <p class="drawer__item-title">Video preview</p>
+                            <iframe
+                                type="text/html"
+                                className="drawer__item"
+                                ng-src="{{capiData.youtubeUrl | wfTrustedUrl}}"
```

Comment:
Good shout. This is more iterative anyway. For self hosted videos, this just wouldn't show